general.rst

```diff
@@ -29,6 +29,7 @@ Procedures
         display_name: the name to display in the user interface e.g. "Quick Realign"
         tool_tip: the tool tip
         messages: any initial messages to display
+        category: Classification of procedures into types, for UI, e.g. 'Action', 'Query', ...  Optional
```

Comment:
How would the UI code use this information ?

Reply:
I got it as the Command-Category from sample_changer.get_availbale_commands():

    def get_available_commands() -> List[List]:
    """
    Retrieves a List (of Lists)  of sample changer specific commands, The List
    has the following structure:

    ["Command-Category_1", [
        ["cmd_1", "DisplayName", "Comment/Help-text"],
        ...
        ["cmd_n-1", "DisplayName", "Comment/Help-text"], 
        ["cmd_n", "DisplayName", "Comment/Help-text"], 
    ] 
    ], 

I assumed that it was needed for setting up the UI, classifying the commands by type, and thought it was better to have the category as a data element than as an organising principle. If it is not needed we should certainly get rid of it.